ui/tests/helpers/module-for-job.js

```diff
@@ -171,7 +175,7 @@ export default function moduleForJob(
       });
 
       test('clicking in a slice takes you to a pre-filtered allocations table', async function (assert) {
-        const slice = JobDetail.allocationsSummary.slices[1];
+        const slice = JobDetail.allocationsSummary.slices[0];
```

Comment:
Note! This was changed from [1] to [0] as we're **now being more strict about what allocations we are creating on a factory-made job** from other improvements in this PR.